include/drivers/sensor.h

```diff
@@ -80,7 +80,16 @@ enum sensor_channel {
 	/** Ambient temperature in degrees Celsius. */
 	SENSOR_CHAN_AMBIENT_TEMP,
 	/** Pressure in kilopascal. */
-	SENSOR_CHAN_PRESS,
+        SENSOR_CHAN_PRESS,
+        /** Euler heading(yawn) in degrees */
+        SENSOR_CHAN_H,
```

Comment:
You need to be more description in the name, I would use the full name here instead, maybe with Euler in the name as well. SENSOR_CHAN_H etc can mean lots of things...

Reply:
Done, sorry, i was a bit fast with the commit :-)